lib/builtin.t

```diff
@@ -347,6 +347,30 @@ package FetchStoreCounter {
     is(prototype(\&builtin::indexed), '@', 'indexed prototype');
 }
 
+# indexed + foreach loop optimisation appears transparent
```

Comment:
commit message "optimisa**s**tion" too many s even for British spelling

Reply:
Ahyes, fixed + forcepushed